<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QScrollBar" name="horizontalScrollBar">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>341</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QScrollBar::horizontal
{
	background-color:rgb(0,255,0);
	margin:5px 50px 5px 50px;
	border:1px transparent;
	border-radius:15px;
}
QScrollBar::handle::horizontal
{
	background-color:rgb(255,0,0);
	border-radius:10px;
}
QScrollBar::sub-line::horizontal
{
	margin:0px 3px 0px 0px;
	height:40px;
	width:40px;
	subcontrol-position:left;
	subcontrol-origin:margin;
}
QScrollBar::sub-line::horizontal:hover, QScrollBar::sub-line::horizontal:on
{
	margin:0px 3px 0px 0px;
	height:40px;
	width:40px;
	subcontrol-position:left;
	subcontrol-origin:margin;
	background-color:rgb(255,0,0);
}
QScrollBar::add-line::horizontal
{
	margin:0px 3px 0px 0px;
	height:40px;
	width:40px;
	subcontrol-position:right;
	subcontrol-origin:margin;
}
QScrollBar::add-line::horizontal:hover, QScrollBar::add-line::horizontal:on
{
	margin:0px 3px 0px 0px;
	height:40px;
	width:40px;
	subcontrol-position:right;
	subcontrol-origin:margin;
	background-color:rgb(255,0,0);
}
QScrollBar::add-page:horizontal,QScrollBar::sub-page:horizontal
{
	background:none;
}
</string>
    </property>
    <property name="maximum">
     <number>500</number>
    </property>
    <property name="singleStep">
     <number>5</number>
    </property>
    <property name="pageStep">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>20</y>
      <width>49</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QScrollBar" name="horizontalScrollBar_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>341</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QScrollBar::horizontal
{
	background-color:rgb(0,255,0);
	margin:5px 50px 5px 50px;
	border:1px transparent;
	border-radius:15px;
}
QScrollBar::handle::horizontal
{
	background-color:rgb(255,0,0);
	border-radius:10px;
}
QScrollBar::sub-line::horizontal
{
	margin:0px 3px 0px 0px;
	height:40px;
	width:40px;
	subcontrol-position:left;
	subcontrol-origin:margin;
}
QScrollBar::sub-line::horizontal:hover, QScrollBar::sub-line::horizontal:on
{
	margin:0px 3px 0px 0px;
	height:40px;
	width:40px;
	subcontrol-position:left;
	subcontrol-origin:margin;
	background-color:rgb(255,0,0);
}
QScrollBar::add-line::horizontal
{
	margin:0px 3px 0px 0px;
	height:40px;
	width:40px;
	subcontrol-position:right;
	subcontrol-origin:margin;
}
QScrollBar::add-line::horizontal:hover, QScrollBar::add-line::horizontal:on
{
	margin:0px 3px 0px 0px;
	height:40px;
	width:40px;
	subcontrol-position:right;
	subcontrol-origin:margin;
	background-color:rgb(255,0,0);
}
QScrollBar::add-page:horizontal,QScrollBar::sub-page:horizontal
{
	background:none;
}
</string>
    </property>
    <property name="maximum">
     <number>500</number>
    </property>
    <property name="singleStep">
     <number>5</number>
    </property>
    <property name="pageStep">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>100</y>
      <width>49</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>horizontalScrollBar</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalScrollBar_2</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>177</x>
     <y>68</y>
    </hint>
    <hint type="destinationlabel">
     <x>181</x>
     <y>148</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalScrollBar_2</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalScrollBar</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>137</x>
     <y>136</y>
    </hint>
    <hint type="destinationlabel">
     <x>142</x>
     <y>61</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
